core/src/main/java/com/hcl/ecomm/core/services/impl/CreateOrderServiceImpl.java

```diff
@@ -118,8 +117,8 @@ public JSONObject createOrderItem(JSONObject orderItem, String cartId, String cu
 					JSONObject jsonRes = new JSONObject();
 					Map emailParams = new HashMap<>();
 					jsonRes = new JSONObject(orderRes);
-					if(jsonRes.length()!=0) {
-						emailParams.put("orderId", order);
+					if(jsonRes.length()!=0 && customerToken != null && !customerToken.isEmpty()) {
```

Comment:
@rishabh-behl  we also require this customerToken to be checked because we need email to be send for logged in user only other wise the code will throw error to guest user's.

Reply:
![successfull](https://user-images.githubusercontent.com/67368779/89919690-3d5cb380-dc19-11ea-9892-5ca32e22df40.PNG)